packages/components/src/button/src/Button.css

```diff
@@ -93,58 +93,90 @@ a.o-ui-button {
     color: var(--hop-primary-text-strong);
 }
 
 /* PRIMARY | HOVER */
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading):hover,
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading).o-ui-button-hover {
     background-color: var(--hop-primary-surface-strong-hover);
+    border-color: var(--hop-primary-surface-strong-hover);
+}
+
+/* PRIMARY | SELECTED / SELECTED + ACTIVE */
+.o-ui-button-primary:not([disabled]):not(.o-ui-button-loading)[aria-pressed="true"],
+.o-ui-button-primary:not([disabled]):not(.o-ui-button-loading)[aria-pressed="true"]:active,
+.o-ui-button-primary:not([disabled]):not(.o-ui-button-loading)[aria-pressed="true"].o-ui-button-active,
+.o-ui-button-primary:not([disabled]):not(.o-ui-button-loading)[aria-checked="true"],
+.o-ui-button-primary:not([disabled]):not(.o-ui-button-loading)[aria-checked="true"]:active,
+.o-ui-button-primary:not([disabled]):not(.o-ui-button-loading)[aria-checked="true"].o-ui-button-active {
+    background-color: var(--hop-primary-surface-selected);
+    border-color: var(--hop-primary-border-selected);
+    color: var(--hop-primary-text-selected);
+}
+
+/* PRIMARY | ACTIVE */
+.o-ui-button-primary:not([disabled]):not(.o-ui-button-loading):active,
+.o-ui-button-primary:not([disabled]):not(.o-ui-button-loading).o-ui-button-active {
+    background-color: var(--hop-primary-surface-strong-press);
+    border-color: var(--hop-primary-surface-strong-press);
+    color: var(--hop-primary-text-strong);
 }
 
 /* PRIMARY | FOCUS */
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading):focus-visible,
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading).o-ui-button-focus {
     background-color: var(--hop-primary-surface-strong-hover);
+    border-color: var(--hop-primary-surface-strong-hover);
 }
 
 /* PRIMARY | HOVER | FOCUS */
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading):hover:focus-visible,
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading).o-ui-button-hover.o-ui-button-focus {
     background-color: var(--hop-primary-surface-strong-hover);
-}
-
-/* PRIMARY | ACTIVE */
-:not([disabled]):not(.o-ui-button-loading).o-ui-button-primary.o-ui-button-active {
-    background-color: var(--hop-primary-surface-strong-selected);
-    outline: 1px solid var(--hop-primary-border-selected);
-    color: var(--hop-primary-text-selected);
+    border-color: var(--hop-primary-surface-strong-hover);
 }
 
 /* UPSELL */
 .o-ui-button-upsell {
     background-color: var(--hop-upsell-surface);
+    border: 1px solid var(--hop-upsell-surface);
     color: var(--hop-upsell-text);
 }
 
 /* UPSELL | HOVER */
 .o-ui-button-upsell:not([disabled]):not(.o-ui-button-loading):hover,
 .o-ui-button-upsell:not([disabled]):not(.o-ui-button-loading).o-ui-button-hover {
     background-color: var(--hop-upsell-surface-hover);
-    outline: 1px solid var(--hop-upsell-border-hover);
+    border-color: var(--hop-upsell-surface-hover);
```

Comment:
normally I'd say put it as the same variable set as the background-color, but variables aren't done the same way as hopper.
It might be better to put a transparent border color here. That way if the background color ever changes, we don't need to update this as well and it's more clear that it's not a mistake.